docs/root/intro/arch_overview/service_discovery.rst

```diff
@@ -102,6 +102,13 @@ used by Envoy to fetch cluster members. Lyft provides a reference implementation
 the backing store, however the API is simple enough that it could easily be implemented on top of a
 variety of different backing stores.
 
+.. attention::
+
+  In the current xDS API, the word "SDS" is used as an acronym for `Secret Discovery Service
+  <https://github.com/envoyproxy/data-plane-api/blob/master/API_OVERVIEW.md#apis>`_.
+
+.. TODO(taiki45): Link to secret discovery service arch overview page when https://github.com/envoyproxy/envoy/pull/4484 gets merged.
```

Comment:
Looks great; do you want to ship now or wait for #4484 to merge?

Reply:
Yes, this PR can wait #4484. Once the PR got merged, I’ll update this PR. Thanks for reviewing.

Reply:
@taiki45 It's been merged so if you want to merge into here we can get this in for 1.8.0. Thank you!

Reply:
Thanks for notifying me. Linked the new SDS doc.